src/client/templates/room.html

```diff
@@ -13,21 +13,31 @@
         <h1 class="text-info">Sprint Retrospective</h1>
         {{> cardSubmitArea}}
         <button class="btn btn-info" id="revealCardButton" type="button">Reveal All Cards</button>
-        <div class="container-fluid">
-            <div class="row">
-                <div class="col-xs-6 col-sm-6">
-                    <h2>Went Well</h2>
-                    {{#each cards "Went Well"}}
-                        {{> card id=_id class="green"}}
+        <div>
+            <table class="table">
```

Comment:
Hmmm, this doesn't seem to put the categories at equal width without a card being added